validator/validators/basic/basic_validator.go

```diff
@@ -116,7 +117,9 @@ func (s *BasicValidator) ValidateLogs(logStream, logLine string, numberOfLogLine
 		}
 		actualNumberOfLogLines := 0
 		for _, l := range logs {
-			if strings.Contains(l, logLine) {
+			if logSource == "WindowsEvents" && logLevel != "" && strings.Contains(l, logLine) && strings.Contains(l, logLevel) {
```

Comment:
Given yours is a feature test and even the config being modified is a feature/windows config, should these changes be in the `feature_validator.go` file instead and leave this `basic_validator.go` unmodified?

Reply:
I don't know anything about this validator code. Can you explain what the `CheckData` function does and why that is a blocker? I agree that this is a specific feature test and don't think this belongs in a "basic" validator

Reply:
CheckData function validates the metrics and logs being emitted and the implementation is available in basic validator.  Feature validator doesn't have the CheckData function and it uses the basic validator checkData function.